docs/modules/publisher-commandline.md

```diff
@@ -379,6 +379,14 @@ The `Alternative` field, where present, refers to the CLI argument applicable in
                                       Mode: Standalone and Orchestrated
                                       Type: boolean
                                       Default: true
+            RejectUnknownRevocationStatus=VALUE
+                                      OPC UA Client Security Config - aReject chain validation 
```

Comment:
`aReject` does not look like a valid word. Maybe the `a` in the font should be removed ?

Reply:
done